tests/components/devolo_home_network/test_switch.py

```diff
@@ -106,18 +106,15 @@ async def test_update_enable_guest_wifi(
     mock_device.device.async_get_wifi_guest_access.return_value = WifiGuestAccessGet(
         enabled=False
     )
-    with patch(
-        "devolo_plc_api.device_api.deviceapi.DeviceApi.async_set_wifi_guest_access",
-        new=AsyncMock(),
-    ) as turn_off:
-        await hass.services.async_call(
-            PLATFORM, SERVICE_TURN_OFF, {"entity_id": state_key}, blocking=True
-        )
+    await hass.services.async_call(
+        PLATFORM, SERVICE_TURN_OFF, {"entity_id": state_key}, blocking=True
```

Comment:
Bonus points if we can use ATTR_ENTITY_ID here

Reply:
I like bonus points ;)